### PR TITLE
Draft PR test

### PR DESCRIPTION
**Pull request by codestreamtest**
Let's see what a draft PR looks like.

### CODE REVIEW

**Comment by codestreamtest** [comment]
Comment in the draft

**Comment by codestreamtest** [comment]
Another comment

File: foo.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { Command, createCommandDecorator, Strings } from "./system";
 const commandRegistry: Command[] = [];
 const command = createCommandDecorator(commandRegistry);
 
+// coldone to test pr desc
```
Comment (planteater) [inline foo.js:16]: Start a review!

Comment (planteater) [reply foo.js:16]: reply to review comment

**Comment by planteater** [comment]
Comment on a PR dude

File: foo.js (hunk 1 above)
Comment (planteater) [inline foo.js:16]: Single comment time.

**Comment by ppezaris** [comment]
test

**Pull request closed without merging**